### ISSUE TITLE
Request for tap-and-hold preview functionality

### ISSUE THREAD

**Issue by theboomr** [zachatrocity/lemmynade#17]
### tap-and-hold preview

I was a user of Joey for Reddit previously, which has a feature where you can tap and hold on an image/gif/video to make it pop up on screen temporarily, and then let go to make the pop up go away again. This was always a really nice usage flow in compact mode. This action would also not mark the post as read either. Joey also had an extra button present on text-only posts in compact mode that you could tap and hold on to preview the beginning of the text post without tapping all the way into it.

**Comment by zachatrocity** [comment]
I may or may not consider this... Especially because now you can just tap the thumbnail now that we have compact view. 

I was going to use the tap and hold on posts to implement a nifty circular/radial menu with some quick actions.

For reference:

https://pub.dev/packages/pie_menu

What are your thoughts on something like that?

**Comment by theboomr** [comment]
I'm not opposed to that idea, as long as it's still at least an option to keep the same immediate buttons present under a post/comment, since personally I prefer the immediacy of being able to tap those buttons vs having to tap and hold to access some particular action. 

And not a big deal if you don't implement the preview idea, I like it but it's probably the least important to me out of the features I've suggested so far!

**Comment by theboomr** [comment]
Also, you could theoretically implement the radial menu with tap and hold on most of the post card and then only do tap and hold preview if it's on the thumbnail itself. But like I said, not a big deal if you don't include it

**Comment by zachatrocity** [comment]
Fair point, the pie menu is kinda over kill. I think you're right more people would use the hold to show image functionality.

Thanks!

**Comment by jordan1776** [comment]
> Fair point, the pie menu is kinda over kill. I think you're right more people would use the hold to show image functionality.

I think the pie menu would be really cool, as long as the hold is fairly short. If there was an option that disabled the "x comments, share, save, vote" bar under posts (maybe move the comment and score to the space near the 3 dot menu) and have those interactions in the pie, this would really differentiate Liftoff from other apps (not just "it's x but for Lemmy")
